src/Elastic.Apm/Api/Request.cs

```diff
@@ -0,0 +1,22 @@
+namespace Elastic.Apm.Api
+{
+	/// <summary>
+	/// Encapsulates Request related information that can be attached to an <see cref="ITransaction" />.
+	/// See <see cref="ITransaction.Request" />
+	/// </summary>
+	public class Request

```

Comment:
We have another internal class called `Request`, but that does not affect users and I think we can live with this.